docs/azure/sdk/resource-management.md

```diff
@@ -74,11 +74,37 @@ using System.Threading.Tasks;
 
 // Code omitted for brevity
 
-var armClient = new ArmClient(new DefaultAzureCredential());
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
 ```
 
 For more information about the `Azure.Identity.DefaultAzureCredential` class, see [DefaultAzureCredential Class](/dotnet/api/azure.identity.defaultazurecredential).
 
+### Management SDK Cheat Sheet
+To quickly get start with our new version Azure management SDK for .NET in less than 5 minutes, imagin you have a task to create/list/update/delete your service bus namespace(a typical kind of Azure resource), below is all the code needed. Just two steps needed.
+First, authenticate and select the subscription & resource group that you want to work on.
+```csharp
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.ServiceBus;
+
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+SubscriptionResource subscription = armClient.GetDefaultSubscription();
+ResourceGroupResource resourceGroupResource = client.GetDefaultSubscription().GetResourceGroup(resourceGroupName).Value;
+```
+Second, find the corresponding method to manage your Azure resource
+| Operation | Method |
+|-|-|
+| Get a resource with resource identifier | armClient.GetServiceBusQueueResource(ResourceIdentifier resourceIdentifier) |
+| List| resourceGroupResource.GetServiceBusNamespaces().GetAll() |
```

Comment:
Well, given the name and the interface indicates, the collection supports to be iterate over - the suffix "collection" and it implements `IEnumerable<T>`, therefore we could simplify this to 
```suggestion
| List| resourceGroupResource.GetServiceBusNamespaces() |
```
which might be more intuitive